conf/application.conf

```diff
@@ -135,6 +135,7 @@ urls {
   exitSurvey = "http://localhost:9514/feedback/plastic-packaging-tax-registration"
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:8505/plastic-packaging-tax/account"
+  mfaUplift = "http://localhost:9553/bas-gateway/uplift-mfa"
```

Comment:
Spot the thing which will need to be pushed out to all the env config...

Reply:
Could just add it to base config repo instead, just like the reg FE